README.md

```diff
@@ -1 +1,101 @@
+
+---
+## Setting up
+To start using **universal_notifications** please add `universal_notifications` to `INSTALLED_APPS` in your Django project, and then migrate the app: `./manage.py migrate universal_notifications`. 
+
+## Basic usage
+* [WebSocket notifications](#websocket-notifications)
+* [E-mail notifications](#e-mail-notifications)
+* [SMS notifications](#sms-notifications)
+* [Push notifications](#push-notifications)
+
+### WebSocket notifications
+``` python
+class OrderShippedWS(WSNotification):
+    message = 'order_shipped'
+    serializer_class = OrderSerializer
+    
+# ... somewhere in a view
+OrderShippedWS(item=order, receivers=[user], context={}).send()
+```
+### E-mail notifications
+``` python
+class OrderShippedEmail(EmailNotification):
+    email_name = 'order_shipped'
+    email_subject = _('{{user.full_name}}, Your order no. {{item.pk}} has been shipped.')
+
+# ... somewhere in a view
+OrderShippedEmail(item=order, receivers=[user], context={'user': user}).send()
+```
+E-mail notifications allow filtering e-mail addreses which are unsubscribed from receiving e-mails from your service. This can be achievied by setting the `UNIVERSAL_NOTIFICATIONS_UNSUBSCRIBED_MODEL` property in your project's settings.
+
+Sample **UnsubscribedModel** should at least contain the **email** property. As result, all addresses existing in **UnsubscribedModel** objects will not receive any e-mails.
+
+### SMS notifications
+``` python
+class OrderShippedSMS(SMSNotification):
+    message = _('{{user.full_name}}, Your order no. {{item.pk}} has been shipped.')
+               
+    def prepare_receivers(self):
+        return {x.shipping_address.phone for x in self.receivers}
+        
+# ... somewhere in a view
+OrderShippedSMS(item=order, receivers=[user], context={'user': user}).send()
+```
+
+This feature requires setting the `UNIVERSAL_NOTIFICATIONS_SMS_FUNC` property in project's settings.
+
+Sample **sms_send** function using [SMSAPI](https://github.com/smsapi/smsapi-python-client)
+``` python
+def send_sms(to_number, text, media=None, priority=9999):
+    api.service('sms').action('send')
+    api.set_content(text)
+    api.set_to(to_number)
+    api.execute()
+```
+
+### Push notifications
+First of all, to use push notifications, you must provide a list of available **devices** linked to users. For more information, please check out [sources](https://github.com/ArabellaTech/universal_notifications/blob/master/universal_notifications/models.py#L20).
+
+Supported platforms:
+* [FCM](https://firebase.google.com/docs/cloud-messaging/) - Android, iOS, Web
+* [GCM](https://developers.google.com/cloud-messaging/) - Android, iOS, Web
+* [APNS](https://developer.apple.com/notifications/) - iOS
+
+To make push notifications work on all supported platforms, a few properties need to be set:
+* UNIVERSAL_NOTIFICATIONS_FCM_API_KEY - Firebase API key
```

Comment:
We don't have this nor one below, all is in `UNIVERSAL_NOTIFICATIONS_MOBILE_APPS`